apps/storybook.namekit.io/stories/Namekit/TruncatedText.stories.tsx

```diff
@@ -48,54 +40,59 @@ type Story = StoryObj<typeof TruncatedText>;
 
 export const ShortText: Story = {
   args: {
-    text: getExampleTruncatedText(TruncatedTextExample.SHORT_TEXT),
+    text: SHORT_TEXT,
+    maxDisplayWidth: 300,
   },
 };
 export const ShortTextStyled: Story = {
   args: {
-    textStylingClasses: "ens-webfont custom-class-name huge-font-size",
-    text: getExampleTruncatedText(TruncatedTextExample.SHORT_TEXT),
+    text: SHORT_TEXT,
+    maxDisplayWidth: 300,
+    textStylingClasses: "ens-webfont colorful-text",
```

Comment:
I didn't like the use of `custom-class-name` because it's too abstract what consequence that has. So I changed it to `colorful-text`.

Also I didn't like the `huge-font-size` examples, they are too extreme.